src/query_cache.js

```diff
@@ -1,12 +1,10 @@
-import hash from "object-hash";
-
 export default class QueryCache {
   constructor() {
     this.cache = {};
   }
 
   getKey(method, url, params) {
-    return method + url + hash(params);
+    return method + url + JSON.stringify(params);
```

Comment:
Note that I changed from a true hash to simply stringifying the params. The external dependency relied on the node `crypto` lib, and also added unnecessary weight to the library size.